doc/modules/ROOT/pages/bookmarks.adoc

```diff
@@ -3,13 +3,15 @@
 Bookmarks ensure that when executing queries in a Neo4j cluster, previous or parallel running queries are fully propagated through the cluster.
 By passing one or multiple bookmarks along with a query, the server will make sure that the query does not get executed before the represented state(s) have been established.
 
-.Example of using bookmarks inside GDS
+.Example of using bookmarks with GDS
 [source, python]
 ----
 # If you ran queries through Neo4j driver before you can pass bookmarks in the constructor
```

Comment:
I am not sure I understand the explanation here. Do you need to specify the `bookmarks=None` parameter here, or is `None` the default and you only need to specify it if you have created any bookmarks with the driver that you want to use?

Reply:
`None` is the default and yes, its if you created a driver instance before and now switch to the GDS client.

Reply:
Ok, so what about rephrasing it as follows?

```suggestion
# Use the `bookmarks` parameter to set any existing bookmarks, for example
# after running a query through the Neo4j driver. The default value is `None`.
```